packages/nimble-components/src/anchor-menu-item/index.ts

```diff
@@ -0,0 +1,125 @@
+import {
+    DesignSystem,
+    AnchorOptions,
+    MenuItem as FoundationMenuItem
+} from '@microsoft/fast-foundation';
+import { styles } from './styles';
+import { template } from './template';
+
+declare global {
+    interface HTMLElementTagNameMap {
+        'nimble-anchor-menu-item': AnchorMenuItem;
+    }
+}
+
+/**
+ * Types of anchor menu item column count.
+ * @public
+ */
+export type AnchorMenuItemColumnCount = 0 | 1;
+
+/**
+ * A nimble-styled anchor menu-item
+ *
+ * This must extend FoundationMenuItem (as opposed to AnchorBase), because the FAST menu has logic
+ * for properly indenting items that only works with FAST menu items.
+ */
+export class AnchorMenuItem extends FoundationMenuItem {
+    @observable
+    public anchor!: HTMLAnchorElement;
+
+    /**
+     * Prompts the user to save the linked URL. See {@link https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a | <a> element } for more information.
+     * @public
+     * @remarks
+     * HTML Attribute: download
+     */
+    @attr
+    public download?: string;
+
+    /**
+     * The URL the hyperlink references. See {@link https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a | <a> element } for more information.
+     * @public
+     * @remarks
+     * HTML Attribute: href
+     */
+    @attr
+    public href?: string;
+
+    /**
+     * Hints at the language of the referenced resource. See {@link https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a | <a> element } for more information.
+     * @public
+     * @remarks
+     * HTML Attribute: hreflang
+     */
+    @attr
+    public hreflang?: string;
+
+    /**
+     * See {@link https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a | <a> element } for more information.
+     * @public
+     * @remarks
+     * HTML Attribute: ping
+     */
+    @attr
+    public ping?: string;
+
+    /**
+     * See {@link https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a | <a> element } for more information.
+     * @public
+     * @remarks
+     * HTML Attribute: referrerpolicy
+     */
+    @attr
+    public referrerpolicy?: string;
+
+    /**
+     * See {@link https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a | <a> element } for more information.
+     * @public
+     * @remarks
+     * HTML Attribute: rel
+     */
+    @attr
+    public rel?: string;
+
+    /**
+     * See {@link https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a | <a> element } for more information.
+     * @public
+     * @remarks
+     * HTML Attribute: target
+     */
+    @attr
+    public target?: '_self' | '_blank' | '_parent' | '_top';
+
+    /**
+     * See {@link https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a | <a> element } for more information.
+     * @public
+     * @remarks
+     * HTML Attribute: type
+     */
+    @attr
+    public type?: string;
+
+    /**
+     * For items in submenus, I could not get clicks to navigate except by using this handler.
+     */
```

Comment:
Work on this phrasing. An arbitrary reader does not know who "I" is.
Also I'd appreciate a more technical analysis than "it doesn't work". I'm very concerned that this link will not behavior like a native link (not just mouse interactions but keyboard interactions and accessibility) and the comment here does not give me confidence expected behaviors will work

Reply:
I've made the comment more detailed. I think we meet the accessibility requirements, but I did find one issue with keyboard interactions. The context menu key brings up the general context menu for the page rather than one for the link. I tried adding logic to handle that by manually sending the keypress to the <a> element, but that only works if I focus the element first, which results in an incorrect outline being drawn. It would be great if I could get delegating focus to work instead, so I may reach out for some help debugging that issue.

Reply:
Looks like ctrl+enter is also behaving inconsistently. In chrome for native `<a>` tags, `<nimble-anchor>`, and `<nimble-anchor-button>` it opens in a new tab as expected. For `anchor-tab` and `anchor-menu` it does not.

Reply:
After discussing offline, I took a different approach involving overriding `setAttribute` so that it forwards `tabindex` to the anchor in the shadow DOM. This avoids the focus loss issue, and with the click and keydown handlers, allows interactions to work as expected (including ctrl + Enter).